src/_pytest/pytester.py

```diff
@@ -1089,14 +1096,42 @@ def collect_by_name(
                 return colitem
         return None
 
+    @overload
     def popen(
         self,
         cmdargs,
-        stdout=subprocess.PIPE,
-        stderr=subprocess.PIPE,
-        stdin=CLOSE_STDIN,
+        stdout: Optional[Union[int, IO]] = subprocess.PIPE,
+        stderr: Optional[Union[int, IO]] = subprocess.PIPE,
+        stdin: Optional[Union[CloseStdinType, bytes, int, IO]] = CLOSE_STDIN,
```

Comment:
Does `Literal[CLOSE_STDIN]` work? If yes, that'd be a little nicer. (I think it is planned to work, but I don't remember if it does already).